scripts/gwf

```diff
@@ -216,6 +220,26 @@ if args.status:
                 print COLORS['red'], task.target.name, CLEAR
             print
 
+elif args.cancel:
+    for target_name in all_targets:
+
+        job_ids = [task.job_id for task in dependencies(workflow.targets, target_name) if task.job_in_queue]
+        if job_ids:
+            print '{}Cancelling job computation of target'.format(COLORS['bold']), target_name, '{}'.format(CLEAR)
+            command = cancel_command(job_ids)
```

Comment:
This only cancels the job -- if the job has produced output files these will exit but (most likely) be corrupt or incomplete -- but probably still cause gwf to consider the job as finished instead of cancelled.

Cancelled jobs should probably be automatically cleaned as well.